build/release/release_npm.cmd

```diff
@@ -0,0 +1,38 @@
+@REM Copyright (c) Microsoft. All rights reserved.
+@REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+@setlocal EnableDelayedExpansion
+@echo off
+
+cd %~dp0\release_npm
+set root=../../..
+
+@REM encourage user to specify a fork of azure-event-hubs
+if "%1"=="" goto usage
+for /f "tokens=1,2,3" %%i in ('git remote -v') do (
```

Comment:
I'm going to assume you know what you're doing here, as my brain is getting a parse error.